### PR TITLE
Fixed the conda syntax because params.enable_conda is deprecated

### PR DESCRIPTION
**Pull request by muffato**
These modules were probably in progress when the overall of the conda syntax happened.

I also fixed the MD5s of the `platypus` tests, that somehow didn't match the current output.

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by muffato** [comment]
The `platypus` check is still failing on Conda. Since it's passing on Docker+Singularity and the files being tested are `.gz` and `.gzi`, I expect it's the usual problem with Conda

**Review by matthdsm — APPROVED** [review]
`conda` still fails for platypus though

File: modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf [hunk 1]
```diff
@@ -3,9 +3,6 @@ process GATK4_DETERMINEGERMLINECONTIGPLOIDY {
     label 'process_single'
 
 
-    if(params.enable_conda){
```
Comment (jfy133) [inline modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf:6]: Does the warning need to be removed entirely, or the condition to check the conda scope

`if (session.config.conda && session.config.conda.enable){`

**Review by jfy133 — COMMENTED** [review]
All lookgs fine except the GATK4 change?

File: modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf (hunk 1 above)
Comment (nvnieuwk) [reply modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf:6]: I'd prefer this indeed

Comment (nvnieuwk) [reply modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf:6]: Oooh awesome! Nvm my comment then 😁

Comment (jfy133) [reply modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf:6]: Ah oops, All good!

**Review by jfy133 — APPROVED** [review]

**Comment by muffato** [comment]
Thank you all !

**Pull request merged**